### PR TITLE
Added tutorial with demo dataset.

### PR DESCRIPTION
**Pull request by k15z**

### CODE REVIEW

File: tutorials/Introducing DataTracer.ipynb [hunk 1]
```diff
@@ -0,0 +1,403 @@
+{
```
Comment (csala) [inline tutorials/Introducing DataTracer.ipynb:1]: It looks like this new tutorial is actually intended to replace the other one, since it basically covers the same functionality with different data and adding a couple new explanations about what's going on.

Can we remove the old one to avoid confusions?

File: tutorials/demo_dataset/make.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import uuid
```
Comment (csala) [inline tutorials/demo_dataset/make.py:1]: Let's remove this module and update the `get_demo_data` from `datatracer/data.py` instead.

Also remove the CSVs from inside the tutorials folder and call the `get_demo_data` function in the tutorial, so users can run `pip install datatrfcer` and directly run the tutorial from any environment without the need to having cloned the repository first.

File: datatracer/foreign_key/standard.py [hunk 3]
```diff
@@ -60,11 +60,11 @@ def fit(self, list_of_databases):
...
-            fks = metadata.get_foreign_keys()
-            fks = set([
-                (fk["table"], fk["field"], fk["ref_table"], fk["ref_field"])
-                for fk in fks
-            ])
+            fks = set()
+            for fk in metadata.get_foreign_keys():
+                if not isinstance(fk["field"], str):
+                    continue  # skip composite keys
+                fks.add((fk["table"], fk["field"], fk["ref_table"], fk["ref_field"]))
```
Comment (csala) [inline datatracer/foreign_key/standard.py:67]: Can we invert this logic?

Reading "If this is a string, add it", is much simpler than "if this is not a string, jump to the next one. If you did not skip the loop at the previous if, add it"

```
            fks = set()
            for fk in metadata.get_foreign_keys():
                if isinstance(fk["field"], str):
                    fks.add((fk["table"], fk["field"], fk["ref_table"], fk["ref_field"]))
```

File: datatracer/data.py [hunk 4]
```diff
@@ -46,10 +47,11 @@ def load_datasets(datasets_path):
         dict:
             Dict of (metadata, tables) tubles, one for each dataset.
     """
-    return {
-        dataset: load_dataset(os.path.join(datasets_path, dataset))
-        for dataset in os.listdir(datasets_path)
-    }
+    datasets = {}
+    for path_to_dataset in glob(os.path.join(datasets_path, "*/")):
```
Comment (csala) [inline datatracer/data.py:51]: Why this change?

File: tutorials/demo_dataset/make.py (hunk 2 above)
Comment (k15z) [reply tutorials/demo_dataset/make.py:1]: Let's circle back on this, we're thinking about replacing this demo dataset entirely with something like https://relational.fit.cvut.cz/dataset/SalesDB.

File: datatracer/foreign_key/standard.py (hunk 3 above)
Comment (k15z) [reply datatracer/foreign_key/standard.py:67]: Sure.

File: datatracer/data.py (hunk 4 above)
Comment (k15z) [reply datatracer/data.py:51]: This was specifically to avoid `.DS_Store` files, but it seems generally reasonable to limit it to folders. It might even make sense to glob for `*/metadata.json` to only select folders that resemble datasets.
